app.go

```diff
@@ -685,10 +677,10 @@ func setGenesisDefaults() {
 	inflation.DefaultInflationState = mintDefaultInflationState()
 }
 
-func slashingDefaultGenesisState() func() *slashingtypes.GenesisState {
-	slashingDefaultGenesisStateFn := slashingtypes.DefaultGenesisState
+func slashingDefaultGenesisState() func() *sdkslashingtypes.GenesisState {
```

Comment:
Being module-specific, shouldn't this be in the x/slashing folder as `func (AppModuleBasic) DefaultGenesis()`. Simlarly for all module default genesis funcs, .i.e, `distrDefaultGenesisState`, staking?

Reply:
I will remove this code. it sets the module default which is set by the module already. 🤷